server/templates/import_wizard.html

```diff
@@ -263,6 +263,8 @@
         <li>Confirm Mappings in the import wizard, one column at a time.
           <ul>
             <li>Identify Place, Variable, Date, and Observation Values Using the Tool</li>
+            <li>For each column title in the uploaded CSV file, the title will either be a specific [Place, Variable, Date, or Observation Value] (option 1 radio button) with corresponding values underneath it OR the column title will specify what the rows underneath it are: either a [Place, Variable, Date, or Observation Value] (option 2 radio button).
```

Comment:
Wonder if its visually easier to parse with bullet points?

e.g.
* Each column title in the uploaded CSV can specify:
   * a specific Place, Variable, Date or Observation Value - Option 1 radio button
   * or, that the rows underneath it are either a Place, Variable, Date or Observation Value - Option 2 radio button

Reply:
done